README.md

```diff
@@ -42,6 +42,11 @@ php composer.phar require chobie/jira-api-restclient ^2.0@dev
 
 * [Composer](https://getcomposer.org/download/)
 
+## Authentication
+
+This client uses basic authentication on the Jira Cloud APIs. Basic authentication requests for Jira Cloud should use [API tokens](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) instead of Atlassian account passwords, as support for basic authentication with passwords has been [deprecated](https://community.developer.atlassian.com/t/announcement-deprecation-of-basic-authentication-with-passwords-and-cookie-based-authentication-in-jira-cloud-rest-apis/15687).
+
+
 ## License
 
-JIRA REST API Client is released under the MIT License. See the bundled [LICENSE](LICENSE) file for details.
+Jira REST API Client is released under the MIT License. See the bundled [LICENSE](LICENSE) file for details.
```

Comment:
Please revert this change, because in here it's library name, not Jira application itself.